docs/dev/commit_the_code_and_submit_a_pull_request.md

```diff
@@ -0,0 +1,35 @@
+# Commit the code and submit a pull request
+TiSpark uses [Git](https://git-scm.com/) to manage its source code. To contribute to the project, you need to get familiar with Git features so that your changes can be incorporated into the codebase.
+This section addresses some of the most common questions and problems that new contributors might face. This section also covers some Git basics; however, if you find that the content is a little difficult to understand, we recommend that you first read the following introductions to Git:
+- The "Beginner" and "Getting Started" sections of this tutorial from [Atlassian](https://www.atlassian.com/git/tutorials)
+- [Documentation](https://docs.github.com/en/get-started/quickstart/set-up-git) and [guides](https://docs.github.com/cn/get-started/using-git/about-git) for beginners from Github
+- A more in-depth [book](https://git-scm.com/book/en/v2/) from Git
+
+## Fork from TiSpark
+First of all, you need to have your own copy of TiSpark. Just click Fork button to get it.
+![frok](pics/fork.png)
+
+## Create your branch
```

Comment:
In my opinion, I can't know where to `checkout`. Maybe we should add `git pull` first and explain that the git command is running on their own PC.
Please make it more detailed, because somebody may use GitHub the first time.